src/DipoleAlgorithms.cc

```diff
@@ -274,107 +212,42 @@ class NaiveDipoleFootprinter : public afw::detection::FootprintFunctor<MaskedIma
 } // anonymous namespace
 
 
-
 /**
  * Given an image and a pixel position, return a Centroid using a naive 3x3 weighted moment
  */
-template<typename PixelT>
-void NaiveDipoleFlux::_apply(
-    afw::table::SourceRecord & source, 
-    afw::image::Exposure<PixelT> const& exposure,
-    afw::geom::Point2D const & center
+void NaiveDipoleFlux::measure(
+    lsst::afw::table::SourceRecord & source, 
+    lsst::afw::image::Exposure<float> const & exposure
 ) const {
-    source.set(getPositiveKeys().flag, true); // say we've failed so that's the result if we throw
-    source.set(getNegativeKeys().flag, true); // say we've failed so that's the result if we throw
+    afw::geom::Point2D center = _centroidExtractor(source, _flagHandler);
```

Comment:
Here's another example where above you expanded to the fully qualified namespace, but leave it relative here.  I may be missing something, but it makes sense to me to try to be as consistent as possible.